int64 ped_id
float64[] belief_value
int64 action